### PR TITLE
User profile and Edit

### PR DESCRIPTION
**Pull request by LongNguyenVu181**
## Related Tickets
https://edu-redmine.sun-asterisk.vn/issues/39845?issue_count=23&issue_position=17&next_issue_id=39843&prev_issue_id=39847

## WHAT
- Create User Profile View
- Edit User Profile

## HOW
- N/A

## WHY
- N/A

## Evidence (Screenshot or Video)
image
image

### CODE REVIEW

**Comment by LongNguyenVu181** [comment]
ready

**Comment by LongNguyenVu181** [comment]
ready

File: hotel/locale/vn/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -41,28 +41,56 @@ msgstr ""
 msgid "Conrfirm Password"
 msgstr ""
```
Comment (huongnt-2545) [inline hotel/locale/vn/LC_MESSAGES/django.po:42]: ủa, ko điền message dịch hả em ??

File: hotel/models.py [hunk 2]
```diff
@@ -2,14 +2,16 @@
...
+from django.urls import reverse
 import uuid
 from phonenumber_field.modelfields import PhoneNumberField
 
 class User(AbstractUser):
     phoneNumber = PhoneNumberField(unique=True)
     USERNAME_FIELD = 'email'
     email = models.EmailField(_('email address'), unique=True)
-    REQUIRED_FIELDS = ['username','phoneNumber']
+    REQUIRED_FIELDS = ['username','phoneNumber']
```
Comment (huongnt-2545) [inline hotel/models.py:13]:
```suggestion
    REQUIRED_FIELDS = ['username', 'phoneNumber']   
```

File: hotel/locale/vn/LC_MESSAGES/django.po (hunk 1 above)
Comment (LongNguyenVu181) [reply hotel/locale/vn/LC_MESSAGES/django.po]: Em đã dịch rồi ạ

**Comment by LongNguyenVu181** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
merged

**Pull request merged**